From 668b45c29c38d440c8fce4bc994c56910adc3919 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20Men=C5=A1=C3=ADk?= <pemensik@redhat.com>
Date: Fri, 14 Dec 2018 17:03:08 +0100
Subject: [PATCH] Fix required tags in few places

Some locations were incorrectly changed to require always tags, else
dnsmasq will not start. Fix dhcp-boot, dhcp-reply-delay and pxe-prompt.
---
 src/option.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/src/option.c
+++ b/src/option.c
@@ -3434,7 +3434,7 @@ static int one_opt(int option, char *arg
       {
 	struct dhcp_netid *id = dhcp_tags(&arg);
 	
-	if (!id)
+	if (!arg)
 	  {
 	    ret_err(gen_err);
 	  }
@@ -3485,7 +3485,7 @@ static int one_opt(int option, char *arg
       {
 	struct dhcp_netid *id = dhcp_tags(&arg);
 	
-	if (!id)
+	if (!arg)
 	  {
 	    ret_err(gen_err);
 	  }
@@ -3515,7 +3515,7 @@ static int one_opt(int option, char *arg
 	 new->opt = 10; /* PXE_MENU_PROMPT */
 	 new->netid = dhcp_tags(&arg);
 	 
-	 if (!new->netid)
+	 if (!arg)
 	   {
 	     dhcp_opt_free(new);
 	     ret_err(gen_err);
